hw/top_earlgrey/ip/ast/doc/ast_regs.html

```diff
@@ -61,10 +61,104 @@
   font-size: 16px;
  }
 </style>
-<table class="regdef" id="Reg_rega0">
```

Comment:
It seems this is not necessarily related to the speedup, but it appears to be an improvement to some of your html colllateral?

Reply:
This is generated by regtool, so I have nothing to change.

Reply:
No problem.